### PR TITLE
macOS use standard toolbar buttons

### LINKED ISSUE #3353: Toolbar icons have wrong mouse highlight
In current nightly if I put mouse over toolbar icon, the icon colour becomes white instead of having the rectangle behindment like other apps. Pause and reset change nothing at all.
This sounds like just a little bug but is I think much aggravating once somebody starts noticing it. I hope toolbar can rewrite with native technologies for next release.
@sweetppro  @Coeur  @nevack

### PR DESCRIPTION
**Pull request by sweetppro**
Fixes #3353

### CODE REVIEW

File: macosx/Controller.mm [hunk 1]
```diff
@@ -5042,12 +5021,27 @@ - (void)updateWindowAfterToolbarChange
...
+        [self hideToolBarBezels:YES];
+
         dispatch_async(dispatch_get_main_queue(), ^{
             [self setWindowSizeToFit];
+            [self hideToolBarBezels:NO];
         });
     }
 }
 
+- (void)hideToolBarBezels:(BOOL)hide
```
Comment (nevack) [inline macosx/Controller.mm:5034]: But why hide them?

Comment (sweetppro) [reply macosx/Controller.mm:5034]: other =wise this occurs #1903

Comment (nevack) [reply macosx/Controller.mm:5034]: But without bezels toolbar looks unnatural :(

Comment (sweetppro) [reply macosx/Controller.mm:5034]: they are restored immediately

Comment (nevack) [reply macosx/Controller.mm:5034]: Ah, okay then

Comment (sweetppro) [reply macosx/Controller.mm:5034]: see here:
```
[self hideToolBarBezels:YES];

        dispatch_async(dispatch_get_main_queue(), ^{
            [self setWindowSizeToFit];
            [self hideToolBarBezels:NO];
        });
```

Comment (sweetppro) [reply macosx/Controller.mm:5034]: more workarounds to fix Cocoa/AppKit bugs...

**Review by nevack — APPROVED** [review]
LGTM

File: macosx/Controller.mm [hunk 2]
```diff
@@ -5042,12 +5021,27 @@ - (void)updateWindowAfterToolbarChange
...
+{
+    if (@available(macOS 11.0, *))
+    {
+        for (NSToolbarItem* item in self.fWindow.toolbar.items)
+        {
+            [item.view setHidden:hide];
```
Comment (DevilDimon) [inline macosx/Controller.mm:5040]:
```suggestion
            item.view.hidden = hide;
```

**Review by DevilDimon — APPROVED** [review]

**Pull request merged**